tests/python/relax/test_codegen_dnnl.py

```diff
@@ -72,6 +72,7 @@ def test_dnnl_offload():
 
     seq = tvm.transform.Sequential(
         [
+            relax.transform.RemovePurityChecking(),
```

Comment:
This is another case where communicating expectations on phase ordering might be helpful. Do we want to deal with `call_pure` in codegen passes and the like? Probably not.